_config.yml

```diff
@@ -63,4 +63,39 @@ defaults:
       path: "products"
     values:
       layout: product
+      releaseCycleColumn: true
+      releaseDateColumn: false
+      discontinuedColumn: false
+      activesupportColumn: false
+      eolColumn: true
+      columns:
+        # The spaces before Yes/No
+        # are intentional to avoid the
+        # YAML norway problem
```

Comment:
A link to https://www.bram.us/2022/01/11/yaml-the-norway-problem/ could be useful for those who don't know what is the "norway problem" ;)